examples/storage_system.rb

```diff
@@ -21,23 +21,50 @@
   password: ''
 }
 
-oneview_storage_system 'Teste' do
+# Example: add storage system or update if it already exists
+oneview_storage_system 'StorageSystem1' do
+  client my_client
   data ({
     credentials:storage_system_credentials,
     managedDomain: 'TestDomain'
   })
-  client client
   action :add
 end
 
-oneview_storage_system 'ThreePAR7200-8147' do
+# Example: add storage system if it does not exist
+oneview_storage_system 'StorageSystem1' do
+  client my_client
   data ({
-    credentials: {
-      ip_hostname: '172.18.11.11',
-      username: 'dcs'
-    },
-    refreshState: 'RefreshPending'
+    credentials:storage_system_credentials,
+    managedDomain: 'TestDomain'
   })
-  client client
-  action :edit
+  action :add_if_missing
+end
+
+# Example: edit storage system credentials
+oneview_storage_system 'StorageSystem1' do
+  client my_client
+  data(
+    ip_hostname: '127.0.0.1',
+    username: 'username',
+    password: 'password'
+  )
+  action :edit_credentials
+end
+
+# Example: edit storage system credentials
```

Comment:
This example is the same as the one before. Isn't this by any chance duplicated?